src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -200,6 +197,24 @@ public PersistentDataStore() {
     return activities;
   }
 
+  public List<Setting> loadSettings()throws PersistentDataStoreException{
```

Comment:
How are we going to make sure that a Setting object is created for all settings we care about?

Reply:
added checking method.